src/App.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import TodoList from './TodoList';
+import AddTodoForm from './AddTodoForm';
+
+function App() {
+  return (
+    <div>
+      <h1>Todo List</h1>
+
```

Comment:
This feels so nitpicky! But do you mind removing the whitespace around the elements. So you have a jsx structure whose formatting is similar to how you've done it in your other components. I think this will make it more readable if this component renders more children.